src/sonic-yang-models/yang-models/sonic-flex_counter.yang

```diff
@@ -161,6 +161,16 @@ module sonic-flex_counter {
                 }
             }
 
+            container FLOW_CNT_TRAP {
```

Comment:
I feel, We can use full name FLOW_COUNTER_TRAP

Reply:
Thanks for the comment. Backend code has been merged, so will keep this name for now.